config/routes.rb

```diff
@@ -322,6 +322,8 @@ def external_redirect(path_prefix, target)
 
         get "/editions/:id" => "editions#show"
 
+        get "/whats-new" => "publisher_updates#index", as: :publisher_updates
```

Comment:
Should we call this controller `whats_new` rather than `publisher_updates`. Having a publisher updates seems to create a second name for the feature which is likely to breed confusion.

Reply:
Good point - the only reason I've kept it with this convention was that the content publisher app calls it `publisher_updates`. I can rename if needed

Reply:
Sure, Content Publisher has a whole controller PublisherInformation rather than a dedicated one for the What's new section. I imagine the action being called publisher_updates in Content Publisher was because this what the page was known as in the team. If we were describing this as the "Publisher update" page this would be a reasonable naming, but since we call it "The what's new page" we should follow that.

Reply:
This has been updated